faker/providers/profile/__init__.py

```diff
@@ -32,7 +33,7 @@ def simple_profile(self):
 
         }
 
-    def profile(self, fields=None):
+    def profile(self, sex=None, fields=None):
```

Comment:
to maintain backward compatibility with code using the arguments positionally, the `sex` argument should be added _after_ `fields`